src/main/scala/org/alvearie/hri/api/MgmtClient.scala

```diff
@@ -18,14 +18,17 @@ import com.fasterxml.jackson.databind.json.JsonMapper
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
 import org.alvearie.hri.flink.core.serialization.NotificationDeserializer
+import org.apache.http.ssl.SSLContexts
+
 import java.util.Base64
 import java.util.ArrayList
 import org.slf4j.LoggerFactory
 
+import java.io.{File, FileNotFoundException}
 import scala.util.{Failure, Success, Try}
 
 class MgmtClient(val baseUri: String, val clientId: String, val clientSecret: String, val audience: String, val oauthServiceBaseUrl: String,
-                 val httpClient: CloseableHttpClient = HttpClients.createDefault()) extends Serializable with BatchLookup {
+                 val httpClient: CloseableHttpClient = MgmtClient.createHttpClient()) extends Serializable with BatchLookup {
```

Comment:
This change at line 31 is Not correct.

See the Private Repo code for `support-2.x` branch here --> https://github.ibm.com/wffh-hri/flink-pipeline-core/blob/support-2.x/src/main/scala/com/ibm/watson/health/foundation/hri/api/MgmtClient.scala

Reply:
I have reverted the `3.x` changes